main.py

```diff
@@ -38,6 +52,127 @@
     db_format = json.load(file_handle)
 
 
+def file_size(file_path):
+    """
+    this function will return the file size
+    """
+    if os.path.isfile(file_path):
+        return os.stat(file_path)
+
+
+def writeToFireBase():
+    """
+    This function will write to Firebase with the given buffer.
+    """
+    try:
+        collections = COL_TELEMETRY.document(timestampStr).collections()
+        for col, sensor in zip(collections, SENSORS):
+            for sec in buffer.keys():
+                data_per_timeframe = int(buffer[sec][sensor])
+                col.document("0").update({
+                    str(sec) : data_per_timeframe
+                })
+        buffer.clear()
+        print("Buffer clear")
+    except Exception as e:
+        exc_type, exc_obj, exc_tb = sys.exc_info()
+        fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
+        print(exc_type, fname, exc_tb.tb_lineno)
+        print(e)
+        
+
+countdownToBufferClear = Timer(60.0, writeToFireBase)
+
+
+
+
+def create():
+    """
+        create() : Add document to Firestore collection with request body
+        Ensure you pass a custom ID as part of json body in post request
+        e.g. json={'id': '1', 'title': 'Write a blog post'}
+    """
+    dateTimeObj = datetime.now()
+    timestampStr = dateTimeObj.strftime("%Y-%m-%d")
+    if not COL_TELEMETRY.document(timestampStr).get().exists:
+        try:
+            COL_TELEMETRY.document(timestampStr).set({"Date": timestampStr})
+            for sensor in SENSORS:
+                COL_TELEMETRY.document(timestampStr).collection(sensor).document("0").set({})
+            return "Documents Created", 201
+        except Exception as e:
+            return f"An Error Occured: {e}", 400
+    return "Document already exists", 200
+
+
+@app.route('/car', methods=['GET', 'POST'])
```

Comment:
GET requests on this endpoint should not be allowed as the code depends on the request body

Reply:
Fixed